File: mobile/app/src/main/java/ru/morozovit/recipeviewer/mobile/Item.java
package ru.morozovit.recipeviewer.mobile;

import android.view.View;

public class Item {
    private final String headline;
    private final String content;
    private final View.OnClickListener onClick;

    public Item(String headline, String content, View.OnClickListener onClick) {
        this.headline = headline;
        this.content = content;
        this.onClick = onClick;
    }

    public String getHeadline() {
        return headline;
    }

    public String getContent() {
        return content;
    }

    public View.OnClickListener getOnClick() {
        return onClick;
    }
}

File: mobile/app/src/main/java/ru/morozovit/recipeviewer/mobile/Item.kt
package ru.morozovit.recipeviewer.mobile

import android.view.View

data class Item(val headline: String, val content: String, val onClick: View.OnClickListener, var selected: Boolean? = false) {
    constructor(headline: String, content: String, onClick: View.OnClickListener): this(headline, content, onClick, false)
}
